jobs/integration/templates/nvidia-smi.yaml

```diff
@@ -1,14 +1,33 @@
-apiVersion: v1
-kind: Pod
+# Reference: https://ubuntu.com/kubernetes/docs/gpu-workers
+apiVersion: batch/v1
+kind: Job
 metadata:
   name: nvidia-smi
 spec:
-  restartPolicy: OnFailure
-  containers:
-  - image: nvidia/cuda:12.1.1-runtime-ubuntu22.04
-    name: nvidia-smi
-    args:
-      - nvidia-smi
-    resources:
-      limits:
-        nvidia.com/gpu: 1
+  template:
+    metadata:
+      name: nvidia-smi
+    spec:
+      restartPolicy: Never
+      containers:
+      - image: nvidia/cuda:12.1.0-base-ubuntu22.04
+        name: nvidia-smi
+        args:
+          - nvidia-smi
+        resources:
+          limits:
+            nvidia.com/gpu: 1
+          requests:
+            nvidia.com/gpu: 1
+        volumeMounts:
+        - mountPath: /usr/bin/
+          name: binaries
+        - mountPath: /usr/lib/x86_64-linux-gnu
+          name: libraries
+      volumes:
+      - name: binaries
+        hostPath:
+          path: /usr/bin/
+      - name: libraries
+        hostPath:
+          path: /usr/lib/x86_64-linux-gnu
```

Comment:
so, this Job needs to mount specific directories?  Can you help me understand the reasons for it?

Reply:
Removed them as not needed